support-frontend/.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
 					},
 				],
 				'@emotion/pkg-renaming': 'error',
+        '@typescript-eslint/no-unused-vars': ["error", { "ignoreRestSiblings": true }],
```

Comment:
this stops eslint complaining with unused vars coming from destructing with the rest operator e.g

if we have

```
const { dontWant, ...doWant } = foo
```

eslint wont complain that we're not using `dontWant`